### PR TITLE
misc: Remove properties from invoice_subscriptions

### PR DESCRIPTION
**Pull request by rsempe**

### CODE REVIEW

File: app/models/invoice_subscription.rb [hunk 1]
```diff
@@ -14,15 +14,13 @@ class InvoiceSubscription < ApplicationRecord
   monetize :total_amount_cents, disable_validation: true, allow_nil: true
 
   scope :order_by_charges_to_datetime,
-        lambda {
-          condition = <<-SQL
-            COALESCE(
-              (invoice_subscriptions.properties->>\'to_datetime\')::timestamp, invoice_subscriptions.created_at
-            ) DESC
-          SQL
+    lambda {
```
Comment (jdenquin) [inline app/models/invoice_subscription.rb:17]: Layout/ArgumentAlignment: Align the arguments of a method call if they span more than one line. (https://rubystyle.guide#no-double-indent)

File: spec/services/invoices/subscription_service_spec.rb [hunk 2]
```diff
@@ -71,10 +71,8 @@
       aggregate_failures do
         expect(result).to be_success
 
-        expect(result.invoice.invoice_subscriptions.first.properties['to_datetime'])
-          .to eq (timestamp - 1.day).end_of_day.as_json
-        expect(result.invoice.invoice_subscriptions.first.properties['from_datetime'])
-          .to eq (timestamp - 1.month).beginning_of_day.as_json
+        expect(result.invoice.invoice_subscriptions.first.to_datetime).to match_datetime((timestamp - 1.day).end_of_day)
+        expect(result.invoice.invoice_subscriptions.first.from_datetime).to match_datetime((timestamp - 1.month).beginning_of_day)
```
Comment (jdenquin) [inline spec/services/invoices/subscription_service_spec.rb:75]: Layout/LineLength: Line is too long. [130/120] (https://rubystyle.guide#max-line-length)

File: app/services/subscriptions/billing_service.rb [hunk 3]
```diff
@@ -242,19 +242,9 @@ def already_billed_today
...
-              CASE WHEN invoice_subscriptions.properties->>'timestamp' ~ '^[0-9\.]+$'
-              THEN
-                -- Timestamp is stored as an integer
-                to_timestamp((invoice_subscriptions.properties->>'timestamp')::integer)::timestamptz
-              ELSE
-                -- Timestamp is stored as a string representing a datetime
-                (invoice_subscriptions.properties->>'timestamp')::timestamptz
-              END
-            )#{at_time_zone(customer: 'cus', organization: 'org')}
+            (invoice_subscriptions.timestamp)#{at_time_zone(customer: 'cus', organization: 'org')}
```
Comment (vincent-pochet) [inline app/services/subscriptions/billing_service.rb:247]: :heart:

**Review by vincent-pochet — APPROVED** [review]
So cool!

**Pull request merged**